src/index.html

```diff
@@ -4,9 +4,446 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="images/logo.png" alt="MOYO_logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__item-link is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item-link" href="#">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item-link" href="#">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item-link" data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item-link" href="#">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item-link" href="#">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item-link" href="#">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item-link" href="#">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div class="card" data-qa="card">
+
+        <img class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27` Retina 5K Monoblock"
+        >
```

Comment:
```suggestion
      <div class="card" data-qa="card">
        <img 
          class="card__image"
          src="images/imac.jpeg"
          alt="APPLE A1419 iMac 27` Retina 5K Monoblock"
        >
```
Not fixed. Also, don't leave empty lines between parent and child elements